packages/@okta/vuepress-site/docs/release-notes/2024/index.md

```diff
@@ -4,6 +4,76 @@ title: Okta Classic Engine API release notes 2024
+
+#### New skipping of entitlement sync during import of a user Systems Log event
+
+A new System Log event has been added to indicate when an entitlement was skipped while importing a user. <!--OKTA-828548-->
+
+#### Create dynamic resource sets with conditions is EA in Production
+
+Resource set conditions help you limit the scope of a role by excluding an admin's access to certain apps. This gives you more granular control over your custom admin roles and helps meet your org's unique security needs. See [Resource set conditions](https://help.okta.com/okta_help.htm?type=oie&id=resource-set-conditions) and the corresponding [Resource Set Resources](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/RoleCResourceSetResource/) API. <!-- OKTA-746719 DYNAMIC_RESOURCE_SETS -->
+
+#### System Log event for emails added to the bounced email list
+
+A System Log `system.email.bounce.removal` event is now triggered when an API request is made to [remove bounced emails](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/EmailCustomization/#tag/EmailCustomization/operation/bulkRemoveEmailAddressBounces) (`POST /org/email/bounces/remove-list`). This request sends a list of emails to a third-party email service to remove the emails from the bounce list. The event is triggered when the API request is made. The event doesn’t indicate when the emails are actually removed by the third-party email service. <!--OKTA-807544 OKTA-754648 -->
+
+#### Granular account linking for certain Identity Providers is self-service EA
+
+When admins link users from SAML and OIDC Identity Providers, they can now exclude specific users and admins. This improves security by allowing admins to configure granular access control scenarios. See Add an External Identity Provider [OpenId Connect](/docs/guides/add-an-external-idp/openidconnect/main/#create-an-identity-provider-in-okta) and [SAML 2.0](/docs/guides/add-an-external-idp/saml2/main/#create-an-identity-provider-in-okta) <!-- EXTENDED_ACCOUNT_LINKING_FILTERS OKTA-831244-->
```

Comment:
```suggestion
When admins link users from SAML and OIDC Identity Providers, they can now exclude specific users and admins. This improves security by allowing admins to configure granular access control scenarios. See [Add an external Identity Provider for OpenId Connect](/docs/guides/add-an-external-idp/openidconnect/main/#create-an-identity-provider-in-okta) and [SAML 2.0](/docs/guides/add-an-external-idp/saml2/main/#create-an-identity-provider-in-okta) <!-- EXTENDED_ACCOUNT_LINKING_FILTERS OKTA-831244-->
```

Reply:
I think this one is formatted this way because there's the link for OpenID Connect and for SAML 2.0 separately. I'm going to leave this one as is.